hack/make-release-artifacts.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e -o pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "${SCRIPTDIR}/.."
+
+bin_dir="out/bin"
+if [[ ! -d "${bin_dir}" ]]; then
+    echo >&2 "Binaries are not built (${bin_dir}), run build-binaries.sh"
+    exit 1
+fi
+
+KREW_ARCHIVE="krew.zip"
+
+echo >&2 "Creating ${KREW_ARCHIVE} archive."
+zip -X -q -r --verbose out/krew.zip "${bin_dir}"
+
+if hash sha256sum 2>/dev/null; then
+  checksum_cmd=sha256sum
+else
+  checksum_cmd="shasum -a 256"
```

Comment:
can we move this before the if, to have `checksum_cmd` in the scope of the document?